app/scripts/components/ticketFinderBox.comp.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+angular.module('yourStlCourts').component('ticketFinderBox', {
+  templateUrl: 'views/ticketFinderBox.html',
+  controller: 'TicketFinderCtrl as ctrl',
+  /*require:{
+    updateFinderSelected: '^HomeCtrl'
+  },*/
+  bindings: {
+    finderSelected: '=',
+    selectFinder: '=',
```

Comment:
You don't need BOTH `finderSelected` and `selectFinder`. By using the `=` in the binding, you're creating a two-way binding. Instead of calling a function callback, just simply assign `ctrl.finderSelected` INSIDE the component and it will bind back to the HomeCtrl which will bind to the other components.